torch/csrc/inductor/aoti_model_runner_cuda.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <torch/csrc/inductor/aoti_model_runner.h>
+
+namespace torch::inductor {
+
+class TORCH_API AOTIModelRunnerCuda : public AOTIModelRunner {
+ public:
+  AOTIModelRunnerCuda(
+      const char* model_path,
+      size_t num_models = 1,
+      const char* cubin_dir = nullptr)
+      : AOTIModelRunner(model_path, num_models, false, cubin_dir) {}
+
+  std::vector<at::Tensor> run(
+      std::vector<at::Tensor> inputs,
+      AOTInductorStreamHandle cuda_stream_handle = nullptr,
```

Comment:
Instead of having the user to pass ```AOTInductorStreamHandle```, I am wondering if it would be better to just let her pass a ```cudaStream_t```. It might be slightly more user-friendly, because the user don't need to worry about what this ```AOTInductorStreamHandle``` is, nor does she need to convert a real ```cudaStream_t``` to a handle. Inside the ```run``` method, we can do the conversion for the user.